IdentityCore/src/oauth2/MSIDWebviewFactory.m

```diff
@@ -261,6 +280,9 @@ - (MSIDAuthorizeWebRequestConfiguration *)authorizeWebRequestConfigurationWithRe
         
     NSString *oauthState = [self generateStateValue];
     NSDictionary *authorizeQuery = [self authorizationParametersFromRequestParameters:parameters pkce:pkce requestState:oauthState];
+#if MSAL_JS_AUTOMATION
+    NSString *javascript = [[parameters allAuthorizeRequestExtraParametersWithMetadata:YES] objectForKey:@"script"];
```

Comment:
nit: let's call script something that for sure will never overlap with any other parameters? e.g. client_automation_script?

Reply:
Made a name change